src/app/relay/mutations/UpdateReportDesignMutation.js

```diff
@@ -36,7 +35,7 @@ class UpdateReportDesignMutation extends Relay.Mutation {
     return {
       id: this.props.id,
       set_fields: JSON.stringify(dynamic.fields),
-      annotation_type: dynamic.annotation_type,
+      annotation_type: 'report_design',
```

Comment:
While we're fixing things ... the next line, `annotated_id`, should not be Stringified.